docs/server.md

```diff
@@ -0,0 +1,90 @@
+# Serving Ollama
```

Comment:
I need to think about this one a bit – i.e. this might be better off being the CLI documentation - of which `ollama serve` is an important comand